bot/seasons/evergreen/fun.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import random
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Fun:
+    """
+    A collection of general commands for fun.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def roll(self, ctx, str_input: str = None):
+        output = ""
+
+        if not str_input:
+            output = "To use .roll, format it as such: .roll (number)"
+        else:
+            try:
+                num_rolls = int(str_input)
```

Comment:
Discord.py supports type [converters](https://discordpy.readthedocs.io/en/rewrite/ext/commands/commands.html#converters) for command parameters. Instead of taking a string and converting it yourself, at the least, you should be annotating the parameter as an `int`.

However, since you wish to restrict the range to be between 1 and 6, you should write a custom type converter. Implementing it that way allows for it to integrate nicely with other parts of the library (such as the error handling).